src/Pagination/Pagination.scss

```diff
@@ -79,6 +94,11 @@
     transition: none;
     line-height: $pagination-line-height;
   }
+
+  &.active .page-link.btn-primary:not(:disabled):not(.disabled):focus {
+    background-color: $pagination-focus-color;
```

Comment:
Should these variables be switched? That is, should `$pagination-bg` be used for `background-color` and vice versa (similar to above)?